src/libraries/System.Net.Http/ref/System.Net.Http.cs

```diff
@@ -148,6 +149,8 @@ public abstract partial class HttpContent : System.IDisposable
         public System.Threading.Tasks.Task LoadIntoBufferAsync(long maxBufferSize) { throw null; }
         public System.Threading.Tasks.Task<byte[]> ReadAsByteArrayAsync() { throw null; }
         public System.Threading.Tasks.Task<byte[]> ReadAsByteArrayAsync(System.Threading.CancellationToken cancellationToken) { throw null; }
+        public System.IO.Stream ReadAsStream() { throw null; }
+        public System.IO.Stream ReadAsStream(System.Threading.CancellationToken cancellationToken) { throw null; }
```

Comment:
Can you please change:
https://github.com/dotnet/runtime/blob/afc9f65bd63f911d9f709fae2bd95021dab65411/src/libraries/System.Net.WebSockets.Client/src/System/Net/WebSockets/WebSocketHandle.Managed.cs#L185
to use ReadAsStream instead of ReadAsStreamAsync?

Do we have a separately work item to move HttpWebRequest's sync APIs to use HttpClient's?

Reply:
WebSocket fixed.
HttpWebRerquest fresh issue here #38441